pom.xml

```diff
@@ -728,7 +728,26 @@
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-site-plugin</artifactId>
-          <version>3.4</version>
+          <version>3.3</version>
```

Comment:
Does the site plugin need to be downgraded to 3.3?
Also it seems like Doxia has a 1.6 version now; might be worth using the latest.
Nit: the indentation of the XML here is deeper than the 2-space indent in the rest of the file.